backend/lucene/src/main/java/org/hibernate/search/backend/lucene/lowlevel/docvalues/impl/MultiValueMode.java

```diff
@@ -0,0 +1,757 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.lucene.lowlevel.docvalues.impl;
+
+import java.io.IOException;
+import java.util.Locale;
+import org.apache.lucene.index.DocValues;
+import org.apache.lucene.index.NumericDocValues;
+import org.apache.lucene.index.SortedDocValues;
+import org.apache.lucene.index.SortedNumericDocValues;
+import org.apache.lucene.index.SortedSetDocValues;
+import org.apache.lucene.search.DocIdSetIterator;
+import org.apache.lucene.util.BitSet;
+import org.apache.lucene.util.BytesRef;
+
+/**
+ * Defines what values to pick in the case a document contains multiple values
+ * for a particular field.
```

Comment:
Same here, please add a few lines to give credit.

Reply:
Corrected